charts/dapr/crds/subscription.yaml

```diff
@@ -51,6 +51,18 @@ spec:
                 type: string
               deadLetterTopic:
                 type: string
+              bulkSubscribe:
+                description: Represents bulk subscribe properies
+                properties:
+                  enabled: 
+                    type: string
+                  maxMessagesCount:
+                    type: string
+                  maxAwaitDurationMs:
+                    type: string
+                required:
+                  - enabled
```

Comment:
Why is this required? Can we assume that it's `false` if unspecified?

Reply:
Basically, `bulkSubscribe` block in itself is optional. So, if it is not required, it shouldn't / wouldn't be given at all.
But, if someone provide `bulkSubscribe` block, then inside it, `enabled` is mandatory.
Does that sound fine?